tsl/test/expected/compression_errors.out

```diff
@@ -782,3 +782,25 @@ SELECT compress_chunk(show_chunks('readings'), TRUE);
 INSERT INTO readings("time", candy, battery_temperature)
     VALUES ('2022-11-11 11:11:11', 33, 0.3)
 ;
+-- Segmentby checks should be done for unique indexes without
+-- constraints, so create a table without constraints and add a unique
+-- index and try to create a table without using the right segmentby
+-- column.
+CREATE TABLE table_unique_index(
+       location smallint not null,
+       device_id smallint not null,
+       time timestamptz not null,
+       value float8 not null
+);
+CREATE UNIQUE index ON table_unique_index(location, device_id, time);
+SELECT table_name FROM create_hypertable('table_unique_index', 'time');
+     table_name     
+--------------------
+ table_unique_index
+(1 row)
+
+\set ON_ERROR_STOP 0
+ALTER TABLE table_unique_index SET (timescaledb.compress);
```

Comment:
We have added a WARNING and not an ERROR. So we don't need this ON_ERROR_STOP?

Reply:
Correct. Fixed that.